TimeSeriesIOTest not passing

```matlab
================================================================================
Verification failed in tests.system.TimeSeriesIOTest/testOutToPyNWB.

    ----------------
    Test Diagnostic:
    ----------------
    
    ======================================================================
    FAIL: testInFromMatNWB (PyNWBIOTest.TimeSeriesIOTest) (container_type='TimeSeries', nwbfield='data')
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/Users/bendichter/dev/matnwb/+tests/+system/PyNWBIOTest.py", line 99, in assertContainerEqual
        self.assertEqual(f1, f2)
    AssertionError: <HDF5 dataset "data": shape (10,), type "<f8"> != [100, 110, 120, 130, 140, 150, 160, 170, 180, 190]
    
    ======================================================================
    FAIL: testInFromMatNWB (PyNWBIOTest.TimeSeriesIOTest) (container_type='TimeSeries', nwbfield='timestamps')
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/Users/bendichter/dev/matnwb/+tests/+system/PyNWBIOTest.py", line 99, in assertContainerEqual
        self.assertEqual(f1, f2)
    AssertionError: <HDF5 dataset "timestamps": shape (10,), type "<f8"> != array([0., 1., 2., 3., 4., 5., 6., 7., 8., 9.])
    
    ----------------------------------------------------------------------
    Ran 1 test in 0.038s
    
    FAILED (failures=2)

    ------------------
    Stack Information:
    ------------------
    In /Users/bendichter/dev/matnwb/+tests/+system/PyNWBIOTest.m (PyNWBIOTest.testOutToPyNWB) at 15
================================================================================
.
================================================================================
Verification failed in tests.system.TimeSeriesIOTest/testInFromPyNWB.

    ----------------
    Test Diagnostic:
    ----------------
    Values for property 'timestamps_interval' are not equal

    ---------------------
    Framework Diagnostic:
    ---------------------
    verifyEqual failed.
    --> Classes do not match.
        
        Actual Class:
            int64
        Expected Class:
            int32
    
    Actual Value:
      int64
    
       1
    Expected Value:
      int32
    
       1

    ------------------
    Stack Information:
    ------------------
    In /Users/bendichter/dev/matnwb/+tests/+system/RoundTripTest.m (RoundTripTest.verifyContainerEqual) at 75
    In /Users/bendichter/dev/matnwb/+tests/+system/PyNWBIOTest.m (PyNWBIOTest.testInFromPyNWB) at 28
================================================================================

================================================================================
Verification failed in tests.system.TimeSeriesIOTest/testInFromPyNWB.

    ----------------
    Test Diagnostic:
    ----------------
    Values for property 'data' are not equal

    ---------------------
    Framework Diagnostic:
    ---------------------
    verifyEqual failed.
    --> Classes do not match.
        
        Actual Class:
            int64
        Expected Class:
            double
    
    Actual Value:
      10×1 int64 column vector
    
       100
       110
       120
       130
       140
       150
       160
       170
       180
       190
    Expected Value:
       100
       110
       120
       130
       140
       150
       160
       170
       180
       190

    ------------------
    Stack Information:
    ------------------
    In /Users/bendichter/dev/matnwb/+tests/+system/RoundTripTest.m (RoundTripTest.verifyContainerEqual) at 75
    In /Users/bendichter/dev/matnwb/+tests/+system/PyNWBIOTest.m (PyNWBIOTest.testInFromPyNWB) at 28
================================================================================
```

--- a/+tests/+system/PyNWBIOTest.py
+++ b/+tests/+system/PyNWBIOTest.py
@@ -4,6 +4,7 @@ import os.path
 import numpy as np
 from dateutil.tz import tzlocal, tzoffset
 import numpy.testing as npt
+import h5py
 
 from pynwb import NWBContainer, get_manager, NWBFile, NWBData, TimeSeries
 from pynwb.ecephys import ElectricalSeries, Clustering
@@ -95,7 +96,7 @@ class PyNWBIOTest(unittest.TestCase):
                     elif isinstance(f2, NWBData):
                         self.assertTrue(np.array_equal(f1.data, f2))
                 else:
-                    if isinstance(f1, (float, np.float32, np.float16)):
+                    if isinstance(f1, (float, np.float32, np.float16, h5py.Dataset)):
                         npt.assert_almost_equal(f1, f2)
                     else:
                         self.assertEqual(f1, f2)

